src/web/interface.rs

```diff
@@ -100,6 +100,23 @@ pub struct ReadResponse<'a> {
     pub instances: HashMap<RbxId, Instance<'a>>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WriteRequest {
+    pub session_id: SessionId,
+    pub removed: Vec<RbxId>,
+
+    #[serde(default)]
+    pub added: HashMap<RbxId, ()>,
```

Comment:
Why this and not HashSet?

Reply:
The `()` is just a placeholder type since most of this struct is unused anyways. Good catch though!